### PR TITLE
Make the status message errors take precedence over action messages

### PR DESCRIPTION
**Pull request by psantos1113**

### CODE REVIEW

File: src/main/js/dynamic/loaded/users/RoleTable.js [hunk 1]
```diff
@@ -292,7 +292,7 @@ RoleTable.propTypes = {
 const mapStateToProps = (state) => ({
     roles: state.roles.data,
     descriptors: state.descriptors.items,
-    roleError: state.roles.roleError,
+    roleError: state.roles.error.message,
```
Comment (psantos1113) [inline src/main/js/dynamic/loaded/users/RoleTable.js:295]: Roles table was not showing forbidden permission message.

File: src/main/js/field/StatusMessage.js [hunk 2]
```diff
@@ -17,8 +17,7 @@ class StatusMessage extends Component {
     componentDidUpdate(prevProps) {
         if (this.props.errorMessage !== prevProps.errorMessage && !this.state.showError) {
             this.setState({ showError: true });
-        }
-        if (this.props.actionMessage !== prevProps.actionMessage && !this.state.showMessage) {
+        } else if (this.props.actionMessage !== prevProps.actionMessage && !this.state.showMessage) {
```
Comment (psantos1113) [inline src/main/js/field/StatusMessage.js:20]: Show either the error message or the action message.  Error message takes precedence.

File: src/main/js/store/actions/roles.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ function fetchedAllRoles(roles) {
 function fetchingAllRolesError(message) {
     return {
         type: USER_MANAGEMENT_ROLE_FETCH_ERROR_ALL,
-        roleFetchError: message
+        message
```
Comment (psantos1113) [inline src/main/js/store/actions/roles.js:33]: Roles table was not showing forbidden permission message.

File: src/main/js/store/actions/users.js [hunk 4]
```diff
@@ -30,7 +30,7 @@ function fetchedAllUsers(users) {
 function fetchingAllUsersError(message) {
     return {
         type: USER_MANAGEMENT_USER_FETCH_ERROR_ALL,
-        userFetchError: message
+        message
```
Comment (psantos1113) [inline src/main/js/store/actions/users.js:33]: user table was not showing forbidden permission message.

File: src/main/js/store/actions/roles.js (hunk 3 above)
Comment (gkillough) [reply src/main/js/store/actions/roles.js:33]: How does this work with no LHS property?

Comment (gkillough) [reply src/main/js/store/actions/roles.js:33]: Also, this is the roles table.

Comment (psantos1113) [reply src/main/js/store/actions/roles.js:33]: Javascript expands the message variable to be` 'message': { message } ` it's shorthand if the LHS matches the variable name.

Comment (psantos1113) [reply src/main/js/store/actions/roles.js:33]: The roles and users table was not displaying the permission denied banner correctly.  It was not setting the correct attributes to be shown in the UI.  That's why the change was made.

**Pull request merged**